nautobot/ipam/models.py

```diff
@@ -114,6 +115,7 @@ class VRF(PrimaryModel):
         verbose_name="Route distinguisher",
         help_text="Unique route distinguisher (as defined in RFC 4364)",
     )
+    status = StatusField(blank=True, null=True)
```

Comment:
I believe this is the first model in core with a nullable Status. Would be good to do some extra testing if so to make sure that this doesn't have unexpected behavior in the UI forms or in the REST API.

Let's also add some `status` values to both `test_views.py` and `test_api.py` to test it as an optional field.